internal/tools/proto-gen-rpc-glue/e2e/source.pb.go

```diff
@@ -0,0 +1,32 @@
+//go:build example
+// +build example
+
+package e2e
+
+import "github.com/hashicorp/consul/proto/pbcommon"
+
+// @consul-rpc-glue: WriteRequest,TargetDatacenter
+type ExampleWriteRequest struct {
+	Value            string
+	WriteRequest     *pbcommon.WriteRequest
+	TargetDatacenter *pbcommon.TargetDatacenter
+}
+
+// @consul-rpc-glue: ReadRequest,TargetDatacenter
+type ExampleReadRequest struct {
+	Value            string
+	ReadRequest      *pbcommon.ReadRequest
+	TargetDatacenter *pbcommon.TargetDatacenter
+}
+
+// @consul-rpc-glue: WriteRequest=AltWriteRequest
+type AltExampleWriteRequest struct {
+	Value           int
+	AltWriteRequest *pbcommon.WriteRequest
```

Comment:
I added this sort of behavior but hadn't used it yet (naming the field something other than the type name).